Uplift "Palo Alto Networks Cortex XDR” Pack to Golden Standard (#37598)
demisto/content: Packs/CortexXDR/ReleaseNotes/6_3_0.md

#### Incident Types

##### Cortex XDR Incident

- Documentation improvement.

##### Cortex XDR Port Scan

- Documentation improvement.

##### Cortex XDR Device Control Violations

- Documentation improvement.

##### Cortex XDR - XCLOUD Cryptomining

- Documentation improvement.

##### Cortex XDR - XCLOUD

- Documentation improvement.

##### Cortex XDR Disconnected endpoints

- Documentation improvement.

##### Cortex XDR - Lite

- Documentation improvement.

#### Integrations

##### Cortex XDR - IOC

- Documentation and metadata improvement.

> - Added support for **Do not use override_severity parameter** parameter which used in order to identify usage of the new configuration of severity param. (%%UPDATE_RN%%)

Reviewer: ```suggestion
- Added support for the **Do not use override_severity parameter** parameter which is used in order to identify usage of the new configuration of the severity param.
```

> - Updated the **API Key ID** parameter to be required.
> - Updated the **API Key** parameter to be required.

Reviewer: ```suggestion
- Updated the *API Key ID* parameter to be required.
- Updated the *API Key* parameter to be required.
```

> - Deprecated and Deleted the ***Override severity*** parameter. Use ***Override severity value*** instead.

Reviewer: ```suggestion
- Deprecated and deleted the *Override severity* parameter. Use *Override severity value* instead.
```

##### Cortex XDR - XQL Query Engine

> - Documentation and metadata improvement.

Reviewer: ```suggestion
Documentation and metadata improvement.
```

##### Palo Alto Networks Cortex XDR - Investigation and Response

- Documentation and metadata improvement.

> - Updated the **API Key ID** parameter to be required.
> - Updated the **API Key** parameter to be required.

Reviewer: ```suggestion
- Updated the *API Key ID* parameter to be required.
- Updated the *API Key* parameter to be required.
```

#### Layouts

##### Cortex XDR - Lite

> - Documentation improvement.
> 
> ##### Cortex XDR Port Scan
> 
> - Documentation improvement.
> 
> ##### Cortex XDR Disconnected endpoints
> 
> - Documentation improvement.
> 
> ##### Cortex XDR Incident
> 
> - Documentation improvement.
> 
> ##### Cortex XDR - XCLOUD
> 
> - Documentation improvement.
> 
> ##### Cortex XDR Device Control Violations
> 
> - Documentation improvement.

Reviewer: ```suggestion
Documentation improvement.

##### Cortex XDR Port Scan

Documentation improvement.

##### Cortex XDR Disconnected endpoints

Documentation improvement.

##### Cortex XDR Incident

Documentation improvement.

##### Cortex XDR - XCLOUD

Documentation improvement.

##### Cortex XDR Device Control Violations

Documentation improvement.
```
